frontend/packages/client/src/components/Community/CommunityEditorDetails/FormConfig.js

```diff
@@ -99,13 +103,13 @@ const AddressSchema = ({ isValidFlowAddress, isEditMode = false } = {}) => {
       .unique('addr', 'Invalid duplicated address'),
     isEditMode
   );
-  return yup.object().shape({
-    ...(isEditMode
-      ? { addrList: elementValidation }
-      : {
-          listAddrAdmins: elementValidation,
-          listAddrAuthors: elementValidation,
-        }),
-  });
+  return yup
+    .object()
+    .shape(
+      Object.assign(
+        {},
+        ...fieldNames.map((fieldName) => ({ [fieldName]: elementValidation }))
+      )
+    );
```

Comment:
using `fieldNames` as param enables having in the same form fields that are arrays like admin addresses and authors then useArrayField will be used for each one of them and `elementValidation` will be applied